qcodes/instrument/visa.py

```diff
@@ -50,46 +57,66 @@ def __init__(self, name, address=None, timeout=5, terminator='', device_clear=Tr
                            vals=vals.MultiType(vals.Numbers(min_value=0),
                                                vals.Enum(None)))
 
+        # backwards-compatibility
+        if address and '@' in address:
+            address, visa_library = address.split('@')
+            if visalib:
+                warnings.warn('You have specified the VISA library in two '
```

Comment:
maybe even error instead of warning? In this case we have a high chance that the driver does not work as expected.

Reply:
I would agree, but I actually don't think that anyone will ever trigger this. I have kept it this way to change the existing tests for the `VisaInstrument` as little as possible.